pkg/frontend/querymiddleware/astmapper/instant_splitting.go

```diff
@@ -194,10 +196,16 @@ func (i *instantSplitter) mapCall(expr *parser.Call) (mapped parser.Expr, finish
 		return i.mapCallAvgOverTime(expr)
 	case countOverTime:
 		return i.mapCallVectorAggregation(expr, parser.SUM)
+	case increase:
+		return i.mapCallVectorAggregation(expr, parser.SUM)
+	case irate:
+		return i.mapCallRate(expr)
 	case maxOverTime:
 		return i.mapCallVectorAggregation(expr, parser.MAX)
 	case minOverTime:
 		return i.mapCallVectorAggregation(expr, parser.MIN)
+	case presentOverTime:
+		return i.mapCallVectorAggregation(expr, parser.MAX)
```

Comment:
Please add a comment to explain why you use MAX. I think it's correct, but just explain it to future readers (and myself too, cause I've a short memory 😉 )